spec/models/reassign_wca_id_spec.rb

```diff
@@ -50,7 +50,6 @@
   end
 
   it "can actually reassign wca id" do
-    team_member = FactoryBot.create(:team_member, user_id: account1.id)
```

Comment:
You should probably add a replacement dummy data that tests reassignment of new Roles instead of old TeamMember

Reply:
Okay done